### PR TITLE
Add spec tests automatically via rake

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

**Comment by MalloZup** [comment]
@juliogonzalez  i was thinking to create automatically the .rspec.yml file , with Repo fetched by the origin branch, but imho is safer on the first manual initialisation, because we cannot be sure what `origin ` is a remote branch 

in this way, we can run automatically the rspec tests
:rocket::

File: doc/TESTS.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ These tests use [rspec](http://rspec.info/) are not executed on travis, so you n
 
 **Run your tests always on your forked repo!**
 
-
+## Syntax variables
```
Comment (juliogonzalez) [inline doc/TESTS.md]: This needs to another hash character so it can be a children of ```## Acceptance tests```

File: doc/TESTS.md [hunk 2]
```diff
@@ -35,6 +35,24 @@ Syntax:
...
 
+
+How to run them:
+
+#### Via rake (best way)
+
+Then create a file called: `.rspec.yml`
+And put your Repo and PR_Number in yml. (this file is ignored by git)
+
+```console
```
Comment (juliogonzalez) [inline doc/TESTS.md]: This not console, but yaml.

File: doc/TESTS.md [hunk 3]
```diff
@@ -35,6 +35,24 @@ Syntax:
...
+repo: MalloZup/gitarro
+pr_num: 1
+```
+
+Then run the tests with
+``` rake spec```
+
```
Comment (juliogonzalez) [inline doc/TESTS.md]: Unneeded extra empty line.

File: doc/TESTS.md [hunk 4]
```diff
@@ -35,6 +35,24 @@ Syntax:
...
+
+#### manually
```
Comment (juliogonzalez) [inline doc/TESTS.md]: ```Manually```

File: doc/TESTS.md (hunk 1 above)
Comment (MalloZup) [reply doc/TESTS.md]: wow beautiful explaination :) +!

File: doc/TESTS.md (hunk 3 above)
Comment (MalloZup) [reply doc/TESTS.md]: fixed

**Comment by MalloZup** [comment]
@juliogonzalez  all your comments adressed

**Comment by juliogonzalez** [comment]
LGTM and WTFB :-)

**Pull request merged**